routes/download.js

```diff
@@ -41,7 +43,9 @@ router.get('/:release_id/:study_id?', (req, res) => {
     })
     .then((obj) => {
       // set response headers
-      res.attachment(`${study_id || release_id}.json.gz`);
+      res.attachment(
+        `${study_id || release_id}.json.${file_format}`,
```

Comment:
is the validation of  file formats taken care by front end?

Reply:
I would imagine that @bdolly provides a radio button, but that's a good point! I will add the validation part.